File: L-MatrisToplama/MatrisToplama.java
import java.util.*;
public class MatrisToplama {
 public static void main(String arg[]) {
  Scanner input = new Scanner(System.in);

  // Değişkenlerin tanımlanması
  int m, n, i, j; 
  System.out.print("Matrislerin satir sayisini giriniz (m)=");
  
  // Boyutların girilmesi
  m = input.nextInt(); 
  System.out.print("Matrislerin sutun sayisini giriniz (n)=");
  n = input.nextInt(); 

  // Matrislerin oluşturulması
  int A[][] = new int[m][n]; 
  int B[][] = new int[m][n];
  int C[][] = new int[m][n];

  // “A” matrisinin girilmesi.
  System.out.println("A matrisini giriniz:");
  for (i = 0; i < m; i++)
   for (j = 0; j < n; j++) {
    System.out.printf("A[%d][%d]=", i + 1, j + 1);
    A[i][j] = input.nextInt();
   } 

  // “B” matrisinin girilmesi. 
  System.out.println("B matrisini giriniz:");
  for (i = 0; i < m; i++)
   for (j = 0; j < n; j++) {
    System.out.printf("B[%d][%d]=", i + 1, j + 1);
    B[i][j] = input.nextInt();
   } 

   // İç içe döngü kullanarak karşılıklı elemanların toplanması.
  for (i = 0; i < m; i++) 
   for (j = 0; j < n; j++) {
    C[i][j] = A[i][j] + B[i][j];
   }
  System.out.printf("A+B=\n");
  // “C” matrisinin yazdırılması.
  for (i = 0; i < m; i++) {
   for (j = 0; j < n; j++)
    System.out.printf("%d ", C[i][j]); 
   System.out.printf("\n");
  }
  input.close();
 }
}

File: L-MatrisToplama/MatrisToplama.cpp
#include <iostream>

using namespace std;

int main() {
    // Değişkenlerin tanımlanması
    int m, n, i, j, ** A, ** B, ** C;

    // Boyutların girilmesi
    cout << "Matrislerin satir sayisini giriniz (m)=";
    cin >> m;
    cout << "Matrislerin sutun sayisini giriniz (n)=";
    cin >> n;

    // Matrislerin oluşturulması
    A = new int * [m];
    B = new int * [m];
    C = new int * [m];
    for (i = 0; i < m; i++) {
        A[i] = new int[n];
        B[i] = new int[n];
        C[i] = new int[n];
    }

    // “A” matrisinin girilmesi.
    cout << "A matrisini giriniz:";
    for (i = 0; i < m; i++)
        for (j = 0; j < n; j++) {
            cout << "A[" << i + 1 << "][" << j + 1 << "]=";
            cin >> A[i][j];
        }

    // “B” matrisinin girilmesi. 
    cout << "B matrisini giriniz:";
    for (i = 0; i < m; i++)
        for (j = 0; j < n; j++) {
            cout << "B[" << i + 1 << "][" << j + 1 << "]=";
            cin >> B[i][j];
        }

    // İç içe döngü kullanarak karşılıklı elemanların toplanması.
    for (i = 0; i < m; i++)
        for (j = 0; j < n; j++) {
            C[i][j] = A[i][j] + B[i][j];
        }
    cout << "A+B=\n";
    // “C” matrisinin yazdırılması.
    for (i = 0; i < m; i++) {
        for (j = 0; j < n; j++)
            cout << C[i][j] << "  ";
        cout << endl;
    }
}